packages/wonder-stuff-ci/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "publishConfig": {
+        "access": "public"
+    },
+    "engines": {
+        "node": ">=16"
+    },
+    "name": "@khanacademy/wonder-stuff-ci",
+    "version": "0.0.1",
+    "description": "Functions for creating automation and scripts.",
+    "scripts": {
```

Comment:
I think you're missing a critical `index.ts` file with all the things that you've created exported. Looking at wonder-stuff-core you'll probably want to add these lines here:

```
    "module": "dist/es/index.js",
    "main": "dist/index.js",
    "types": "dist/index.d.ts",
```

And then make a `src/index.ts` file that imports your files and re-exports them again.